crypto/evp_extra/evp_extra_test.cc

```diff
@@ -3402,3 +3402,81 @@ TEST(EVPExtraTest, RawKeyUnsupported) {
       EVP_PKEY_new_raw_private_key(EVP_PKEY_RSA, nullptr, kKey, sizeof(kKey)));
 }
 
+TEST(EVPExtraTest, DSATrailingSignatureGarbage) {
```

Comment:
It would be good to add comments to the test case on what fails and why.

Reply:
Is the `EXPECT_FALSE` and on calls to verify a test case name not sufficient. What can use clarification?

Reply:
Added a comment above the signature value explaining that it has extra bytes after the DER encoding.

Reply:
Thanks. Yes, that's right the `EXPECT_FALSE` is clear. The return values of `DSA_check_signature` and `DSA_verify` are what I needed to ensure they're indicating failures, so I checked them in the code.